cont/LuaUI/actions.lua

```diff
@@ -178,98 +185,104 @@ function actionHandler:RemoveWidgetActions(widget)
   clearActionList(self.keyReleaseActions)
 end
 
-
 --------------------------------------------------------------------------------
 --------------------------------------------------------------------------------
 --
 --  Calls
 --
 
-local function MakeWords(line)
-  local words = {}
-  for w in string.gmatch(line, "[^%s]+") do
-    table.insert(words, w)
+-- Split a string into a table of substrings, based on a delimiter.
+-- If not supplied, delimiter defaults to whitespace.
+-- Consecutive delimiters are treated as one.
+-- string.split(csvText, ',')	csvText:split(',')
+local function ssplit(val, delimiter)
```

Comment:
Why rename the function (or why touch it at all if the only difference is an optional arg that never gets used)?

Reply:
True, maybe unnecessary, although I'd have preferred to have a `string.split` defined instead of using this. Anyway, I'll revert it back with a more sensible name than `MakeWords`